### PR TITLE
Update upcoming events, and add to bird pages

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Description
_The commit messages say what you did; this should explain why and/or how._

- update upcoming talks
- add upcoming talks to bird pages, because there was nowhere to see a list

We have a larger story to think through how we show events long-term, but this seemed like a quick solution to feature upcoming events on my bird page.

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- /authors/miriam/
- /talks/css-is-rad/

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screen Shot 2020-07-23 at 12 50 18 PM
Screen Shot 2020-07-23 at 12 50 51 PM

### CODE REVIEW

File: content/_includes/page/list.njk [hunk 1]
```diff
@@ -9,6 +9,11 @@
     collections
   ) }}
 {% elif bird and (bird != 'oddbird') %}
+  {{ confs.list(
+    title='Upcoming Events',
+    events=collections.all | byBird(bird) | getData('data.events'),
+    all=false
+  ) }}
```
Comment (jgerigmeyer) [inline content/_includes/page/list.njk:16]: @mirisuzanne This adds an "Upcoming Events" title to all Bird pages, even if there are no events:

Screenshot 2020-07-23 15 06 42

File: content/_includes/events.macros.njk [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{% import 'layout.macros.njk' as layout %}
```
Comment (jgerigmeyer) [inline content/_includes/events.macros.njk:1]: This file is missing a global `{# @docs ... #}` header with `label` etc.

File: content/_includes/events.macros.njk [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          {{ utility.link_if(event.venue, event.url) }}{% if event.adr %} {{ '--' | typogr | safe }}
+          {{ event.adr }}{% endif %}
+        </li>
+      {%- endfor -%}
+    </ol>
+  {%- endif -%}
+{% endmacro %}
+
+{% macro datetime(event) -%}
+  <time datetime="{{ event.date | getDate('iso') }}">{{ event.date | getDate('short') }}</time>
```
Comment (jgerigmeyer) [inline content/_includes/events.macros.njk:33]: Doesn't need to hold up merge, but I cringe a bit at "Jun" as a shortened version of "June". I'd like to replace our `formatDate` filter with an actual date formatting library, e.g. date-fns.

**Comment by mirisuzanne** [comment]
@jgerigmeyer I think I addressed all your notes here, aside from the new stories you created

**Pull request merged**